main.js

```diff
@@ -2,6 +2,23 @@ const addBtn = document.getElementById("addNote");
 const notesContainer = document.getElementById("notesContainer");
 const noNotesImg = document.querySelector(".no-notes-img");
 
+let currentItemCount=()=>{
+    const allNotes = document.querySelectorAll(".note");
+    let notes = [];
+    let ele = {};
+
+    allNotes.forEach((note) => {
+        ele.content = note.querySelector(".noteContent").value;
+        ele.title = note.querySelector(".inputHeading").value;
+        if (!(ele.content === "" && ele.title === "" || ele.content === " " && ele.title === " ")) {
+            notes.push(ele);
+        }
+        ele = {};
+    })
+
+    return notes.length
+}
+
```

Comment:
1. Return `notes` from the function and reuse this function inside `updateLSData` to fetch all notes.
2. change the function name to something like `getAllNotes`
3. Use the `notes.length` property while fetching the length of the notes or create a separate function to return the length by passing `notes` as the parameter.